iXBRLViewerPlugin/viewer/src/js/util.js

```diff
@@ -32,6 +32,7 @@ export const STORAGE_APP_LANGUAGE = "ixbrl-viewer-app-language";
 export const STORAGE_THEME = "ixbrl-viewer-theme";
 export const STORAGE_HIGHLIGHT_FACTS = "ixbrl-viewer-highlight-all-facts";
 export const STORAGE_HOME_LINK_QUERY = "ixbrl-viewer-home-link-query";
+export const USER_GUIDE_URL = "https://arelle-ixbrl-viewer.readthedocs.io/";
```

Comment:
Should this be https://arelle-ixbrl-viewer.readthedocs.io/en/latest/user_guides/user_guide.html (or even `__VERSION__`)?  It seems unfortunate to link to the root, which has all the building/installing/etc. information, which doesn't seem useful for most users.